### DISCUSSION TITLE
Bugged Projectile

### DISCUSSION THREAD

**Discussion by collinistevens** [Minestom/Minestom#928] [Help] [answered]
Occasionally when shooting a projectile it will remain on the ground for the client even if removed from the instance. The entity cannot be seen by others players and is fixed by relogging.

**Comment by collinistevens** [comment, marked as answer]
Issue is fixed - Minestom version was a few days out of date.

**Reply by TheMode** [reply]
This should have been reported as an issue, not help discussion.
